applications/GeoMechanicsApplication/custom_elements/plane_strain_stress_state.cpp

```diff
@@ -0,0 +1,64 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+#include "plane_strain_stress_state.h"
+#include "custom_utilities/stress_strain_utilities.hpp"
+
+namespace Kratos
+{
+
+Matrix PlaneStrainStressState::CalculateBMatrix(const Matrix&         rGradNpT,
+                                                const Vector&         rNp,
+                                                const Geometry<Node>& rGeometry) const
+{
+    const auto dimension       = rGeometry.WorkingSpaceDimension();
+    const auto number_of_nodes = rGeometry.size();
+    Matrix     result = ZeroMatrix(VOIGT_SIZE_2D_AXISYMMETRIC, dimension * number_of_nodes);
+
+    for (unsigned int i = 0; i < number_of_nodes; ++i) {
+        const auto offset = dimension * i;
+
+        result(INDEX_2D_PLANE_STRAIN_XX, offset + INDEX_X) = rGradNpT(i, INDEX_X);
+        result(INDEX_2D_PLANE_STRAIN_YY, offset + INDEX_Y) = rGradNpT(i, INDEX_Y);
+        result(INDEX_2D_PLANE_STRAIN_XY, offset + INDEX_X) = rGradNpT(i, INDEX_Y);
+        result(INDEX_2D_PLANE_STRAIN_XY, offset + INDEX_Y) = rGradNpT(i, INDEX_X);
+    }
+
+    return result;
+}
+
+double PlaneStrainStressState::CalculateIntegrationCoefficient(const Geometry<Node>::IntegrationPointType& rIntegrationPoint,
+                                                               double detJ,
```

Comment:
Nitpicking (to follow the Kratos Style Guide):
```suggestion
                                                               double DetJ,
```

Reply:
Done